package.json

```diff
@@ -29,7 +29,6 @@
     "@ng-select/ng-select": "^3.0.7",
     "@ctrl/ngx-codemirror": "^2.1.0",
     "@ngx-progressbar/core": "^5.3.2",
-    "angular-plotly.js": "^1.3.2",
```

Comment:
@mshariq-nerd  we need to remove this from https://github.com/foglamp/foglamp-gui/blob/develop/docs/developers-guide.md /cc @monika-Dianomic